src/components/Popups/index.tsx

```diff
@@ -59,6 +60,9 @@ export default function Popups() {
   // get all popups
   const activePopups = useActivePopups()
 
+  // show survey popup if user has not closed
+  const [showSurveyPopup] = useShowSurveyPopup()
```

Comment:
seems a bit like a smell to have to grab the popup state from outside the popup 🤔

Reply:
Yeah i think this could be best fixed in a survey manager component - the dynamic width that this is being used for does feel kinda weird